typescript/sdk/src/consts/multisigIsm.ts

```diff
@@ -439,10 +439,6 @@ export const defaultMultisigConfigs: ChainMap<MultisigConfig> = {
     ],
   },
 
-  eclipse: {
```

Comment:
how come?

Reply:
it was a duplicate entry, see https://github.com/hyperlane-xyz/hyperlane-monorepo/pull/4359/files#diff-69567aec038546b64be29d4a095e0707d7e0b82ff4696c32945c74458023adf3L145

Reply:
https://y.yarn.co/a026b9b9-2104-4b07-aade-afe5ee6d7674_text.gif